include/dlr_treelite.h

```diff
@@ -80,6 +83,8 @@ class DLR_DLL TreeliteModel : public DLRModel {
   virtual void Run() override;
   virtual void SetNumThreads(int threads) override;
   virtual void UseCPUAffinity(bool use) override;
+
+  inline void SetPredMargin() { pred_margin = 1; };
```

Comment:
I think we should allow user to enable and disable, could you add a boolean arg to `SetPredMargin()`?

Reply:
Will do